app/src/main/resources/config/application-local-dev.yml

```diff
@@ -70,6 +70,7 @@ newsfeeds:
 
 googleAnalyticsAccount:
     main: UA-55852460-3
+    portfolio: UA-55852460-16 # hei tässä on QA:n avain, muuta devin avaimeksi
```

Comment:
Ei kai haittaa että dev ja QA käyttävät samaa GA-tunnusta? OO toimi aikaisemminkin näin. Voi poistaa kommentin tai tehdä tiksun erillisen dev-tunnuksen hankkimisesta. (kommentit lisäksi englanniksi jos niille on tarvetta)